CONTRIBUTING.md

```diff
@@ -90,6 +90,25 @@ completed checklist.
 The following set of guidelines must be adhered to when opening pull requests in the Azure
 PowerShell repository.
 
+#### Target Release Types
+
+Here are all types of releases of **Azure PowerShell**. Note that the type of release isn't necessarily aligned to the state of the swagger or service feature.
+
+- **General** release
+  - This PR targets an [official general release](https://github.com/Azure/azure-powershell/milestones) of Azure PowerShell.
+  - Target version is `x.y.z`, no suffix.
+  - Target branch should be `main` or `generation` depending on the development approach.
+- **Preview** release
```

Comment:
For preview release, shall we include the case for 0.x.y releases?

Reply:
Good point. Updated.